towhee/runtime/data_queue.py

```diff
@@ -156,7 +194,17 @@ def type_schema(self) -> List[str]:
             >>> dq.type_schema
             [<ColumnType.SCALAR: 2>, <ColumnType.QUEUE: 1>]
         """
-        return list(self._schema.col_types())
+        return self._schema.col_types
+
+    def _get_size(self):
+        for index in self._scalar_index:
```

Comment:
we don't need to check this condition every time after all the scalar value being set